eng/resolveContract.targets

```diff
@@ -60,11 +61,36 @@
                                       ReferenceAssembly="" />
     </ItemGroup>
   </Target>
-  
+
+  <Target Name="GenerateDocIdsForCoreLibOrUriProject"
```

Comment:
I need to add a better name/comment for this:

What this does is it takes the SPC or Uri implementation assembly and generates a .txt with all public types in it.

That file is then later on passed to GenAPI as the list of types which should be "split out" with the conditional compilation logic.